ui/src/common/consts.js

```diff
@@ -0,0 +1,2 @@
+let LOSSLESS_FORMATS = ['FLAC', 'WAV', 'DSF']
+export default LOSSLESS_FORMATS
```

Comment:
We have a `ui/src/consts.js` file. These should be there, not in a new `consts.js` file (we only need one)